web/src/pages/apd/activities/cost-allocation/CostAllocateFFPQuarterly.test.js

```diff
@@ -422,13 +532,31 @@ describe('the cost allocation quarterly FFP component', () => {
       screen.getByRole('textbox', {
         name: 'federal share for ffy 2022, quarter 1, state'
       }),
-      '5'
+      '57'
+    );
+
+    expect(
+      screen.getByRole('textbox', {
+        name: 'federal share for ffy 2022, quarter 1, state'
+      })
+    ).toHaveValue('57');
+
+    await user.clear(
+      screen.getByRole('textbox', {
+        name: 'federal share for ffy 2022, quarter 1, state'
+      })
+    );
+    await user.type(
```

Comment:
you should do this with those values above and just have them blank to start or like 25 or something, whatever it looked like when it was throwing the error

Reply:
on 492 and 498 :)